packages/core/hooks/useFileAccessContextMenu.ts

```diff
@@ -136,7 +136,7 @@ export default (filters?: FileFilter[], onDismiss?: () => void) => {
                         ],
                     },
                 },
-                ...(isQueryingAicsFms
+                ...(isQueryingAicsFms && isOnWeb
```

Comment:
are we preventing editing on desktop?

Reply:
Yea I was originally but I think I'll add it back and use LK or something to grab the passwords - I just didn't want the desktop to be a relic holding stale passwords

Reply:
Now that the stuff is in the bucket this now allows editing on desktop too